docs/getting-started/introduction.mdx

```diff
@@ -16,7 +16,7 @@ import '@tldraw/tldraw/tldraw.css'
 
 export default function () {
 	return (
-		<div style={{ position: 'fixed', inset: 0 }}>
+		<div className="tldraw__editor">
```

Comment:
@steveruizok do we need to update this example to match the `BasicExample` like this? Or can we leave it as-is?